<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="6">
     <widget class="QTableWidget" name="pointsTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>281</width>
        <height>351</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>281</width>
        <height>351</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>21</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>41</number>
      </attribute>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="6" rowspan="12">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="xLbl">
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="xSB">
      <property name="maximum">
       <number>1184</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="yLbl">
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QSpinBox" name="ySB">
      <property name="maximum">
       <number>873</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="addPointBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Добавить
вершину</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="4">
     <widget class="QPushButton" name="setSeedPointBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Установить
затравочную
точку</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="6">
     <widget class="QPushButton" name="deletePointBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Удалить последнюю
вершину</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="6">
     <widget class="QPushButton" name="closeFigBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Замкнуть фигуру</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLabel" name="colorLbl">
      <property name="text">
       <string>Цвет заполнения:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="2">
     <widget class="QPushButton" name="colorBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="delayLbl">
      <property name="text">
       <string>Задержка:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="3">
     <widget class="QSpinBox" name="delaySB">
      <property name="maximum">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="msLbl">
      <property name="text">
       <string>мс</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="6">
     <widget class="QPushButton" name="paintBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Закрасить</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="6">
     <widget class="QPushButton" name="clearBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="6">
     <widget class="QPushButton" name="roolsBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Правила ввода</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="seedLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Затравка:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="4">
     <widget class="QLabel" name="seedStateLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>не установлена</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="6">
     <widget class="QLabel" name="timeLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Время заполнения:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
